addons/mail/static/src/components/activity/activity.xml

```diff
@@ -116,10 +118,10 @@
                     </t>
 
                     <t t-if="activityView.activity.canWrite">
-                        <div name="tools" class="o_Activity_tools">
+                        <div name="tools" class="o_Activity_tools d-flex">
                             <t t-if="!activityView.fileUploader">
                                 <Popover position="'right'" title="activityView.markDoneText">
-                                    <button class="o_Activity_toolButton o_Activity_markDoneButton btn btn-link" t-att-title="activityView.markDoneText">
+                                    <button class="o_Activity_toolButton o_Activity_markDoneButton btn btn-link btn-primary pt-0 pl-0" t-att-title="activityView.markDoneText">
```

Comment:
Aren't `btn-link` and `btn-primary` "conflicting" style classes?

Reply:
> Aren't btn-link and btn-primary "conflicting" style classes?

Nope, we handle this combination https://github.com/odoo/odoo/blob/master/addons/web/static/src/webclient/webclient.scss#L143

Note: we'll review the way how we handle this kind of buttons, but it will be after the migration.